zero/meta-arducam/conf/layer.conf

```diff
@@ -6,6 +6,7 @@ BBFILES += "\
     ${LAYERDIR}/recipes-devicetree/linux-raspberrypi_%.bbappend \
```

Comment:
I don't know where to put my comment on the commit log so here goes.
- please check english grammer.
- The arducam_ppa provides packages and there is a package providing a pre-built "binary".
- The word "various" isn't the right word because you are installing `.so` and `.hpp`.  You should generate the symlink.
- this "Libcamera's build system utilizes the first pkgconfig to search for this SDK, so it also installs .pc file." isn't right.  libcamera uses pkgconfig (or pkg-config).  pkgconfig uses the first matched `.pc` file (if it's true; I haven't tested).
- This "It used the following page as a reference for how to install pre-built packages with Yocto." is unbigous.  Since we were just talking about "Libcamera's build system", "pkgconfig", or the `.pc` file, the word "it" sounds like referencing one of them.  Instead I'm sure you are referencing the `.bb` file.

Reply:
my take:

> zero/meta-arducam: Install libarducam_pivariety.so.1.0.2 from Arducam SDK
> 
> This commit adds pre-built libarducam_pivariety.so taken from Arducam SDK.
> 
> Arducam has extended libcamera source code and has published it on
> GitHub. However, the source code needs libarducam_pivariety.so and
> it's header file to build and run.
> 
>   - https://github.com/ArduCAM/arducam_ppa
>     arducam-pivariety-sdk-dev
> 
> (We should add SHA hash or tags where we obtained files)
> (We should add list of files taken from the package)
> 
> Libcamera build system utilizes pkgconfig and pkgconfig utilizes the
> first .pc, we are adding the .pc file to -dev package as well.
> 
> The arducam-pivariety_1.0.2.bb was writen referencing the following
> page: "Working with Pre-Built Libraries"
> 
>   - https://docs.yoctoproject.org/dev/dev-manual/prebuilt-libraries.html

Please update hash value, tags, package name, list of files taken.

Reply:
Thanks. will update commit log and add SHA hash.etc.